cmd/skopeo/utils.go

```diff
@@ -143,6 +149,10 @@ func (opts *imageOptions) newSystemContext() (*types.SystemContext, error) {
 		DockerDaemonCertPath:     opts.dockerCertPath,
 		SystemRegistriesConfPath: opts.global.registriesConfPath,
 	}
+	// replace the shared authfile with the specific one if present.
```

Comment:
(Absolutely non-blocking: not sure the comment is necessary, to an extent just repeats what the code says.)

Reply:
removed